src/clj/game/cards/icebreakers.clj

```diff
@@ -709,17 +709,15 @@
    "Ika"
    (auto-icebreaker ["Sentry"]
                     {:abilities [(break-sub 1 2 "Sentry")
-                                 (strength-pump 2 3 :all-run)
+                                 (strength-pump 2 3)
                                  {:label "2 [Credits]: Host Ika on a piece of ICE"
```

Comment:
The cost will now automatically be added to the subroutine label because of the `:cost` vector, so we can remove it from this string.

Reply:
Aha, meaning I can just remove the first part of the `:label`?